CI is broken, cannot build source dist for OBS

**Describe the bug**
I tried to run the CI step to push to OBS and I expected to update the package instead it failed

**GEOPM version**
Output from `geopmread --version`
593caaa

**Error messages produced**
```
Warning, treated as error:
autodoc: failed to import module 'io' from module 'geopmpy'; the following exception was raised:
No module named 'pandas'
make[2]: *** [Makefile:3876: docs/build/man/.dirfile] Error 2
make[2]: Leaving directory '/home/runner/work/geopm/geopm/service'
make[1]: *** [Makefile:3185: distdir] Error 2
make[1]: Leaving directory '/home/runner/work/geopm/geopm/service'
make: *** [Makefile:3262: dist] Error 2
Error: Process completed with exit code 2.
```

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -44,7 +44,7 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: install system dependencies
-      run: sudo apt-get update && sudo apt-get install libelf-dev mpich libmpich-dev libomp-11-dev libsystemd-dev python3-gi
+      run: sudo apt-get update && sudo apt-get install libelf-dev mpich libmpich-dev libomp-11-dev libsystemd-dev python3-gi python3-yaml
     - name: install base and service dir python dependencies
       run: |
            python3 -m pip install --upgrade pip setuptools wheel pep517
@@ -102,7 +102,7 @@ jobs:
       with:
         fetch-depth: 0
     - name: install system dependencies
-      run: sudo apt-get update && sudo apt-get install libsystemd-dev libgirepository1.0-dev osc python3-m2crypto
+      run: sudo apt-get update && sudo apt-get install libsystemd-dev osc python3-m2crypto python3-gi python3-yaml
     - name: install base and service dir python dependencies
       run: |
            python3 -m pip install --upgrade pip setuptools wheel pep517

--- a/service/docs/source/conf.py
+++ b/service/docs/source/conf.py
@@ -76,7 +76,9 @@ source_suffix = '.rst'
 napoleon_google_docstring = True
 
 autodoc_mock_imports = ['geopmdpy.gffi',
-                        'pandas']
+                        'pandas',
+                        'numpy',
+                        'natsort']
 
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ['_templates']

